Detections/MultipleDataSources/MultiplePasswordresetsbyUser.yaml

```diff
@@ -91,11 +99,14 @@ query: |
   | extend ResetPivot = "TotalUserReset")
   )
   | extend timestamp = StartTimeUtc, AccountCustomEntity = Account, HostCustomEntity = Computer
+  | extend Name = tostring(split(AccountCustomEntity,'@',0)[0]), UPNSuffix = tostring(split(AccountCustomEntity,'@',1)[0])
 entityMappings:
   - entityType: Account
     fieldMappings:
-      - identifier: FullName
+      - identifier: Name
         columnName: AccountCustomEntity
```

Comment:
1. This should be Name.
2. Also, line 101 is not required. We can directly use Account and Computer.
3. For Host also, map Name and NTDomain/DnsDomain mapping. FullName is deprecated.
4. TargetUserName also requires split by @